src/app/write/components/PreventModal/PreventModal.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Modal } from '@/components';
+import { useOverlay } from '@/hooks';
+import { useRouter } from 'next/navigation';
+import { useCallback } from 'react';
+
+interface ModalProps {
+  onClose: () => void;
+}
+
+const PreventModal = ({ onClose }: ModalProps) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [openModal, closeModal] = useOverlay();
```

Comment:
배열 구조분해시에는 변수명 생략하는 것을 통해서 에러없이 처리 가능해요~

```suggestion
  const [, closeModal] = useOverlay();
```